includes/class-wc-admin-api-init.php

```diff
@@ -555,10 +616,10 @@ public static function queue_dependent_action( $action, $prerequisite_action ) {
 			self::queue()->schedule_single(
 				$after_blocking_job,
 				self::QUEUE_DEPEDENT_ACTION,
-				array( $action, $prerequisite_action )
+				array( $action, $action_args, $prerequisite_action )
 			);
 		} else {
-			self::queue()->schedule_single( time() + 5, $action );
+			self::queue()->schedule_single( time() + 5, $action, $action_args );
```

Comment:
I see we use `time() + 5` above as well.  Just curious if there's any significance here behind the 5 seconds.

Reply:
Just following the example set by @peterfabian here: https://github.com/woocommerce/wc-admin/pull/1050/files#diff-760a9caa82f609ceea4369cbff49237fR233